### ISSUE TITLE
Getting Different Results when Reproducing AGNews with Singular Prototypical Verbalizer

### ISSUE THREAD

**Issue by harshkaria** [thunlp/OpenPrompt#257]
Hi,

I am trying to reproduce AGNews using a single verbalizer following #228. My YAML is as follows:
```
dataset:
  name: agnews
  path: datasets/TextClassification/agnews

plm:
  model_name: roberta
  model_path: roberta-large
  optimize:
    freeze_para: False
    lr: 0.00003
    weight_decay: 0.01
    scheduler:
      type: 
      num_warmup_steps: 500

checkpoint:
  save_latest: False
  save_best: False

train:
  batch_size: 2
  num_epochs: 5
  train_verblizer: alternate
  clean: True

test:
  batch_size: 2

template: manual_template
verbalizer: proto_verbalizer

manual_template:
  choice: 0
  file_path: scripts/TextClassification/agnews/manual_template.txt

proto_verbalizer:
  parent_config: verbalizer
  choice: 0
  file_path: scripts/TextClassification/agnews/manual_verbalizer.txt
  lr: 0.01
  mid_dim: 128
  epochs: 30
  multi_verb: proto

  

environment:
  num_gpus: 1
  cuda_visible_devices:
    - 0
  local_rank: 0
  
learning_setting: few_shot

few_shot:
  parent_config: learning_setting
  few_shot_sampling: sampling_from_train
  
sampling_from_train:
  parent_config: few_shot_sampling
  num_examples_per_label: 1
  also_sample_dev: True
  num_examples_per_label_dev: 1
  seed:
    - 123

reproduce:  # seed for reproduction 
  seed: 123  # a seed for all random part
```

I switched mid_dim from 64 to 128 as described in the ProtoVerb paper, but I am only getting an accuracy of 46.1% when in the paper it is reported an accuracy of 81.65% for k=4.

**Comment by harshkaria** [comment]
Closing this issue, have to change `num_examples_per_label` from `1` to `4` as the paper reflects k as the number of examples per label, not the total number of samples

**Comment by czlll** [comment]
Hi @harshkaria , did you get the 81.65% accuracy as reported in the paper with k=4?  I only get an accuracy of 74.1% when I set `num_examples_per_label: 4` and `mid_dim: 128`.

**Comment by harshkaria** [comment]
> 
@czlll 
We were able to get 85.71% on AGNews with those parameters. Trained on Google Colab.
